website/src/components/Navbar.vue

```diff
@@ -1,16 +1,18 @@
 <template>
   <chakra.nav
     h="60px"
-    w="100vw"
+    w="100%"
     px="4"
     d="flex"
     position="fixed"
     align-items="center"
     shadow="sm"
     bg="white"
     z-index="10"
+    border-top="6px solid"
```

Comment:
I really liked this. 😄 Nice job!

Reply:
Was inspired by the Chakra UI (react) site 💪